sdk/anomalydetector/ai-anomaly-detector-rest/src/pollingHelper.ts

```diff
@@ -1,68 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import { Client, HttpResponse } from "@azure-rest/core-client";
```

Comment:
@qiaozha @joheredi FYI

I notice that the LRO flag is removed in this pr: https://github.com/Azure/azure-rest-api-specs/pull/21410 and relevant Cadl issue is tracked here: https://github.com/Azure/azure-rest-api-specs/issues/21527.

Reply:
From the issue, it looks like the change is expected.